Gemfile

```diff
@@ -2,7 +2,7 @@ source "https://rubygems.org"
 gemspec
 
 group :development, :test do
-  gem "omnibus", github: "opscode/omnibus"
+  gem "omnibus", github: "chef/omnibus", branch: "ksubrama/gcc_investigate"
```

Comment:
I think this branch _is_ needed currently. At least for the [`to_msys2_path`](https://github.com/chef/omnibus/pull/695/files#diff-54da3d3ae44ba0a204d406bf64f8fd37R286) method [used in `liblzma.rb`](https://github.com/chef/omnibus-software/pull/711/files#diff-3de7b12049016c9be3ae20d815a451adR45).

I needed ...